src/briefcase/platforms/web/static.py

```diff
@@ -0,0 +1,347 @@
+                        "pack",
+                        self.app_path(app),
+                        "--dest-dir",
+                        self.wheel_path(app),
+                    ],
+                    check=True,
+                )
+            except subprocess.CalledProcessError as e:
+                raise BriefcaseCommandError(
+                    f"Unable to build wheel for app {app.app_name!r}"
+                ) from e
+
+        with self.input.wait_bar("Installing wheels for dependencies..."):
+            try:
+                self.tools.subprocess.run(
+                    [
+                        sys.executable,
+                        "-u",
+                        "-m",
+                        "pip",
+                        "wheel",
+                        "--wheel-dir",
+                        self.wheel_path(app),
+                        "-r",
+                        self.bundle_path(app) / "requirements.txt",
+                    ],
+                    check=True,
+                )
+            except subprocess.CalledProcessError as e:
+                raise BriefcaseCommandError(
+                    f"Unable to install dependencies for app {app.app_name!r}"
+                ) from e
+
+        with self.input.wait_bar("Writing Pyscript configuration file..."):
+            with (self.project_path(app) / "pyscript.toml").open(
+                "w", encoding="utf-8"
+            ) as f:
+                f.write(f"name = '{app.formal_name}'\n")
+                f.write(f"description = '{app.description}'\n")
+                f.write(f"version = '{app.version}'\n")
+                f.write("\n")
+                f.write("autoclose_loader = true\n")
+                f.write("packages = [\n")
+                f.write(
+                    "\n".join(
+                        f'    "/{wheel.relative_to(self.project_path(app))}",'
+                        for wheel in self.wheel_path(app).glob("*.whl")
+                    )
+                )
+                f.write("\n]\n")
+
+        self.logger.info("Compile static web content from wheels...")
+        with self.input.wait_bar("Compiling static web content from wheels..."):
+            # Trim previously compiled content out of briefcase.css
+            briefcase_css_path = (
+                self.project_path(app) / "static" / "css" / "briefcase.css"
+            )
+            self._trim_file(
+                briefcase_css_path,
+                sentinel=" ******************* Wheel contributed styles **********************/",
+            )
+
+            # Extract static resources from packaged wheels
+            for wheelfile in self.wheel_path(app).glob("*.whl"):
+                self.logger.info(f"Processing {wheelfile.name}...")
+                with briefcase_css_path.open("a", encoding="utf-8") as css_file:
+                    self._process_wheel(wheelfile, css_file=css_file)
+
+        return {}
+
+
+class HTTPHandler(SimpleHTTPRequestHandler):
```

Comment:
Not a bad idea; although it might be worth treating that as a separate feature request if it turns out to be needed.

Reply:
As a demonstration, attempting to use the web server while switching between two apps in the same project fails because the browser uses the cached `pyscript.toml` and tries to pull the wheel for the other app.

Of course, holding SHIFT while clicking refresh is enough to force the browser to flush the cache.

Given that, this is only mildly convincing to me we should introduce complications to address this now...especially with a trivial (and normal web dev) workaround. Example for posterity, nonetheless.